app/models/info_request.rb

```diff
@@ -1456,14 +1456,24 @@ def applicable_law
   end
 
   def title_formatting
-    if title && !MySociety::Validate.uses_mixed_capitals(title, 1)
+    return unless title
+    unless MySociety::Validate.uses_mixed_capitals(title, 1) ||
+      title_starts_with_number || title_is_acronym(6)
       errors.add(:title, _('Please write the summary using a mixture of capital and lower case letters. This makes it easier for others to read.'))
     end
-    if title && title =~ /^(FOI|Freedom of Information)\s*requests?$/i
+    if title =~ /^(FOI|Freedom of Information)\s*requests?$/i
       errors.add(:title, _('Please describe more what the request is about in the subject. There is no need to say it is an FOI request, we add that on anyway.'))
     end
   end
 
+  def title_is_acronym(length)
+    title.upcase == title && title.length < length && !title.include?(" ")
```

Comment:
although maybe I should have rewritten it as 

```
def title_is_acronym(allow_shorter_than)
```

and kept the original less than to match 

```
def self.uses_mixed_capitals(s, allow_shorter_than = 20)
```

from commonlib/validate

Left for a future revision